src/aleph/sdk/client/abstract.py

```diff
@@ -75,25 +76,35 @@ async def get_posts(
         :param post_filter: Filter to apply to the posts (Default: None)
         :param ignore_invalid_messages: Ignore invalid messages (Default: True)
         :param invalid_messages_log_level: Log level to use for invalid messages (Default: logging.NOTSET)
+        :param verify_signatures: Whether to verify the signatures of the messages (Default: False)
```

Comment:
Would it be more readable in the imperative form ?
```suggestion
        :param verify_signatures: Verify the signatures of the messages (Default: False)
```